packages/shared/src/components/post/PostModalActions.tsx

```diff
@@ -0,0 +1,135 @@
+import React, {
+  CSSProperties,
+  KeyboardEventHandler,
+  MouseEventHandler,
+  ReactElement,
+  useContext,
+  useState,
+} from 'react';
+import classNames from 'classnames';
+import dynamic from 'next/dynamic';
+import MenuIcon from '../../../icons/menu.svg';
+import CloseIcon from '../../../icons/x.svg';
+import OpenLinkIcon from '../../../icons/open_link.svg';
+import { Roles } from '../../lib/user';
+import AuthContext from '../../contexts/AuthContext';
+import { Post } from '../../graphql/posts';
+import useNotification from '../../hooks/useNotification';
+import useReportPostMenu from '../../hooks/useReportPostMenu';
+import classed from '../../lib/classed';
+import { SimpleTooltip } from '../tooltips/SimpleTooltip';
+import { CardNotification } from '../cards/Card';
+import { Button } from '../buttons/Button';
+import PostOptionsMenu from '../PostOptionsMenu';
+import AnalyticsContext from '../../contexts/AnalyticsContext';
+import { postAnalyticsEvent } from '../../lib/feed';
+import { PostOrigin } from '../../hooks/analytics/useAnalyticsContextData';
+
+export interface PostModalActionsProps {
+  post: Post;
+  onClose?: MouseEventHandler | KeyboardEventHandler;
+  className?: string;
+  style?: CSSProperties;
+  inlineActions?: boolean;
+  origin?: PostOrigin;
+}
+
+const Container = classed('div', 'flex flex-row items-center');
```

Comment:
Not breaking, but perhaps we should consider making these classes generic at one stage

Reply:
Totally agree. We probably need to come up with a better naming on that one.